src/main/java/at/hannibal2/skyhanni/features/garden/visitor/VisitorListener.kt

```diff
@@ -125,42 +128,56 @@ class VisitorListener {
         inventory.handleMouseClick_skyhanni(slot, slot.slotIndex, 0, 0)
     }
 
+    @SubscribeEvent
+    fun onBackgroundDrawn(event: GuiContainerEvent.BackgroundDrawnEvent) {
+        if (!VisitorAPI.inInventory) return
+        if (!config.rewardWarning.preventRefusing && !config.rewardWarning.preventRefusingCopper && !config.rewardWarning.preventAcceptingCopper) return
+        if (config.rewardWarning.bypassKey.isKeyHeld()) return
+
+        val visitor = VisitorAPI.getVisitor(lastClickedNpc) ?: return
+        val refuseOfferStack = event.gui.inventorySlots.getSlot(33).stack ?: return
+        val acceptOfferStack = event.gui.inventorySlots.getSlot(29).stack ?: return
+
+        if (visitor.hasReward() != null && config.rewardWarning.preventRefusing)
+            refuseOfferStack.background = LorenzColor.DARK_GRAY.addOpacity(180).rgb
+        if (visitor.pricePerCopper <= config.rewardWarning.coinsPerCopperPrice && config.rewardWarning.preventRefusingCopper)
+            refuseOfferStack.background = LorenzColor.DARK_GRAY.addOpacity(180).rgb
+        if (visitor.pricePerCopper >= config.rewardWarning.coinsPerCopperPrice && config.rewardWarning.preventAcceptingCopper)
+            acceptOfferStack.background = LorenzColor.DARK_GRAY.addOpacity(180).rgb
```

Comment:
dont like the formatting of these if statements

Reply:
wdym?

Reply:
Either have them be one line or have brackets not